init.lua

```diff
@@ -360,6 +360,15 @@ require('lazy').setup({
 
   { -- Fuzzy Finder (files, lsp, etc)
     'nvim-telescope/telescope.nvim',
+    -- To disable the default Telescope plugin and replace it with
+    -- another picker (like snacks), set enabled to false and
+    -- Enable your replacement picker by requiring it explicitly (e.g., 'kickstart.plugins.snacks')
+    -- By default, Telescope is included and acts as your picker for everything.
+
+    -- Note: When you customize your config for yourself,
+    -- it’s best to remove the Telescope plugin config entirely
+    -- instead of just disabling it here, to keep your config clean.
```

Comment:
I do agree it might encourage snacks a bit too much.\
what do you think about this?
 ```suggestion
    -- By default, Telescope is included and acts as your picker for everything.

    -- If you would like to switch to a different picker (like snacks, or fzf-lua)
    -- you can disable the Telescope plugin by setting enabled to false and enable
    -- your replacement picker by requiring it explicitly (e.g. 'custom.plugins.snacks')

    -- Note: If you customize your config for yourself,
    -- it’s best to remove the Telescope plugin config entirely
    -- instead of just disabling it here, to keep your config clean.
```

Reply:
For line 363-364, I think removing code blocks for plugin configs should be more encouraged when replacing same feature plugins, instead of recommending toggling an `enabled` key and keeping stale configs. In fact, this is what the note in line 368 says, and if the configs are already laid out intuitively (i.e. features cleanly separated), then that should also be unneeded.

For line 365, there's already documentation for this in line 983 onwards in current `init.lua`, specifically about importing custom.plugins, and `custom/plugins/init.lua`.

Line 366 can be shortened and merged instead with the top description in line 361.